### PR TITLE
Bitmark SDK: Complete Edit

### PR DESCRIPTION
**Pull request by shannona**

### CODE REVIEW

**Comment by vienlt** [comment]
@nodestory: Please help to look at this PR when you have time. Thank you.

**Comment by vienlt** [comment]
@nodestory, @jollyjoker992, @anhnguyenbitmark: Please spend time review this Pr, it is the line editing for SDK documents.

**Review by anhnguyenbitmark — APPROVED** [review]

File: pages/bitmark-references/bitmark-sdk/README.md [hunk 1]
```diff
@@ -4,74 +4,75 @@
...
 
-
-## [Store Seed](store-seed.md#store-seed)
-  
-* [Android](store-seed.md#android)
-* [iOS](store-seed.md#ios)
-
-
-## [Web Socket](websocket.md#web-socket)
+## [Triggering from Web Socket Events](websocket.md#web-socket)
```
Comment (nodestory) [inline pages/bitmark-references/bitmark-sdk/README.md]:
```suggestion
## [Subscribing to events via WebSocket](websocket.md#web-socket)
```

The server triggers events to notify clients about the updates on the blockchain. For SDK users, the action should be about how to get the events sent from the server.

**Review by nodestory — COMMENTED** [review]
Every part looks good! The only thing I'm not sure is about the description `Triggering from Web Socket Events`.

The server triggers events to notify clients about the updates on the blockchain. For SDK users, the action should be about how to get the events sent from the server, i.e. **receiving events** or **subscribing to events**.

**Comment by shannona** [comment]
@nodestory Would Receiving Web Socket Events work?

**Comment by nodestory** [comment]
Yes! So it will be `Receiving WebSocket Events`. There is no space between Web and Socket.

**Comment by vienlt** [comment]
@shannona, @nodestory: Do we need any other work on this PR?

**Comment by shannona** [comment]
I've made the change @nodestory and I discussed. So: good for me.

**Review by nodestory — APPROVED** [review]

**Pull request merged**